std/digest/digest.d

```diff
@@ -774,7 +774,8 @@ string toHexString(Order order = Order.increasing, LetterCase letterCase = Lette
         }
     }
     import std.exception : assumeUnique;
-    return assumeUnique(result);
+    // memory was just created, so casting to immutable is safe
+    return () @trusted { return assumeUnique(result); }();
```

Comment:
Spec [says](https://dlang.org/spec/function.html#function-safety) that "explicit casting of mutable objects to immutable" is forbidden, but is my assumption correct that we can allow the `cast` safely if we have created the memory ourselves?